packages/taquito-contracts-library/package.json

```diff
@@ -68,6 +68,7 @@
     ]
   },
   "dependencies": {
+    "@taquito/core": "^16.1.2-beta-RC.0",
```

Comment:
I think master is already at 16.1.2, could you merge master into this branch for the latest versions?

Reply:
will do